### PR TITLE
Train a linear classifier on Iris dataset

### PR DESCRIPTION
**Pull request by Milkve**
Iris dataset is a Hello-World dataset on machine learning, now you can use `tf_iris.ipynb`, `tf_iris.py` and `rtt_iris.py` to learn RTT in short time. Some preprocess on Iris dataset is in `tf_iris.ipynb`.

p.s. Due to RTT cannot support some operation like `tf.ones_like()`, P0 and P1 the two parties must move training label and test label mean value to 0 locally and respectively. You can find steps in `tf_iris.ipynb`.

### CODE REVIEW

File: example/tutorials/code/rtt_iris.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+EPOCHES = 1
+BATCH_SIZE = 32
+learning_rate = 0.003
+
+rtt.activate("SecureNN")
+mpc_player_id = rtt.py_protocol_handler.get_party_id()
+
+# real data
+# ######################################## difference from tensorflow
+training_features_file_path = "/home/juzix/RosettaPlayground/example/tutorials/dsets/P" + str(mpc_player_id) + "/iris_training_features.csv"
```
Comment (vingstar) [inline example/tutorials/code/rtt_iris.py:25]: DO NOT use absolute path of your local machine in source code.

File: example/tutorials/code/tf_iris.ipynb [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "looking-watts",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# #############################################################################\n",
+    "# # This method of data loading is not suitable for static graph in TF1.14, \n",
```
Comment (vingstar) [inline example/tutorials/code/tf_iris.ipynb:63]: I don't understand this, any more explanation?

**Review by vingstar — CHANGES_REQUESTED** [review]
Dear Milkve, 
Thanks for your contribution!
In my opinion, it may be better to move this example into another new directory, such as `contrib`, and make it more self-contained.

File: example/tutorials/code/rtt_iris.py (hunk 1 above)
Comment (Milkve) [reply example/tutorials/code/rtt_iris.py:25]: OK

File: example/tutorials/code/tf_iris.ipynb (hunk 2 above)
Comment (Milkve) [reply example/tutorials/code/tf_iris.ipynb:63]: These lines are prepared for future Rosetta which may support dynamic graph. The dynamic graph can tell users more about what they are doing in data and model dimensions. Now many popular python libs provide with APIs to load classic datasets like Iris, and the up-to-date tutorials of loading Iris in TF is eager mode loading.
Many teachers in their ML and DL courses recommend students to deduce the flows of tensor on scratch paper. Noobies will refer to latest official tutorials and they use eager mode data loading as TF official say. These line can remind noobies of avoiding them concat code and wheels roughly from others.

File: example/contrib/rtt_iris.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    # Y_pred_cls0 = np.where(Y_pred >= 0, Y_pred, -1)
+    # Y_pred_cls0 = np.where(Y_pred < 0, Y_pred_cls0, 0)
+    # Y_pred_cls2 = np.where(Y_pred < 0.05, Y_pred, 1)
+    # Y_pred_cls2 = np.where(Y_pred >= 0.05, Y_pred_cls2, 0)
+    # Y_pred_cls012 = Y_pred_cls0 + Y_pred_cls2
+    # Y_comp = np.hstack((Y_pred_cls012, test_label))
+    # print(Y_comp)
+    # acc = len([i for i in Y_pred_cls012 if i in test_label]) / len(test_label)
+    # print('acc: ', acc)
+
```
Comment (vingstar) [inline example/contrib/rtt_iris.py:106]: You may also evaluate the result in MPC style, please refer to our current examples.

File: example/tutorials/code/rtt_iris.py (hunk 1 above)
Comment (vingstar) [reply example/tutorials/code/rtt_iris.py:25]: These codes under the `code` directory should be removed since there has been a copy in the directory `contrib`.

**Review by vingstar — CHANGES_REQUESTED** [review]

**Review by vingstar — APPROVED** [review]
LGTM
